checkov/kubernetes/checks/graph_checks/ModifyServicesStatus.yaml

```
@@ -0,0 +1,61 @@
+metadata:
+  id: "CKV2_K8S_4"
+  name: "SAs and nodes that can modify services/status may set the `status.loadBalancer.ingress.ip` field to exploit 
```

```suggestion
  name: "ServiceAccounts and nodes that can modify services/status may set the `status.loadBalancer.ingress.ip` field to exploit 
```
no need for abbreviation here 😄